.travis.yml

```diff
@@ -17,6 +17,7 @@ before_install:
 install:
   - docker build -t savaslabs/squabble:latest .
   - make install
+  - composer install
```

Comment:
I would prefer for `composer` operations to happen within the container, so that we are not mixing up operations on the host and in the container